extensions/sql-database-projects/src/models/connections/connectionService.ts

```diff
@@ -79,6 +79,7 @@ export class ConnectionService {
 				multipleActiveResultSets: undefined,
 				packetSize: undefined,
 				persistSecurityInfo: undefined,
+				hostNameInCertificate: undefined,
```

Comment:
Not yet, it's planned currently for Decemeber, but I can fast track the changes there to get compatibility there if needed.
What do you think should we check-in to vscode mssql extension first or here?

Reply:
It would probably be worth checking if you even need to do anything on the VS Code MSSQL side first - hopefully it's just passing the details directly to STS and so wouldn't actually require any changes in the extension itself (except for updating STS)

And then even if you don't add this right now it just means that the value will be ignored right? So that can be something that's documented as a known issue - it doesn't seem like a huge blocker to most people anyways for this specific scenario.

Reply:
As long as there aren't any errors with vscode-mssql not recognizing this new property, then it's fine to update here first. But if there's a risk of this causing problems if vscode-mssql doesn't also get updated, I would prefer to wait for this change to happen in the sql projects extension after code complete for November so it would only get included in the December release.

Reply:
I'm fine delaying this as well, since this is a new feature and not urgently requested or anything. Also discussed with @kburtram internally, we can take this in December release as we'll have better compatibility with the extension.